sensor/AdblockPlugin.js

```diff
@@ -269,17 +292,21 @@ class AdblockPlugin extends Sensor {
       });
     }
 
-    async cleanUpFilter() {
-      const config = await this.getAdblockConfig();
-      for (const key in config) {
-        const file = `${dnsmasqConfigFolder}/${key}_adblock.conf`;
-        try {
-          if (fs.existsSync(file)) {
-            await fs.unlinkAsync(file);
+    _cleanUpFilter(config) {
+      try {
+        const result = []
+        if (typeof config == 'object') {
+          for (const key in config) {
+            if(config[key] == "on") result.push(key+"_adblock.conf")
```

Comment:
how to make sure any leftover configs are removed as well?

1. enabled list ad1 and ad2
2. ad1 is removed from cloud
3. when next cleanup filter, will ad1 list be removed from the dnsmasq config directory?